filesystem/src/main/java/org/frankframework/filesystem/FileSystemActor.java

```diff
@@ -555,7 +555,7 @@ public void setInputFolder(String inputFolder) {
 	}
 
 	/**
-	 * If set to <code>true</code>, if a folder is part of the fileName, this will be created inside the root folder for the file to create, write, move or copy if it does not exist
+	 * If <code>true</code>: if a non-existing folder is part of the fileName, it will be created inside the root folder. Side-effect: value of <code>inputFolder</code> is checked when a message passes the pipe, not during startup.
```

Comment:
Ik weet niet of het er nu duiderlijker of onduidelijker van wordt. Het `root` attribuut bestaat alleen in de `LocalFileSystemPipe`. Parameters worden altijd runtime uitgelezen. Ik denk dat jij hier de parameter bedoeld want het attribuut wordt in `FileSystemActor#open()` uitgelezen.